pkg/testcase/service.go

```diff
@@ -81,14 +81,14 @@ func TestServiceLoadBalancer(applyWorkload, deleteWorkload bool) {
 	Expect(err).NotTo(HaveOccurred(), err)
 
 	getAppLoadBalancer := "kubectl get pods -n test-loadbalancer  " +
-		"--field-selector=status.phase=Running --kubeconfig="
+		"--field-selector=status.phase=Running "
```

Comment:
on some setups kubectl may not work without providing the kubeconfig. double checking removing this is essential here.

Reply:
this is only used here, and its used in  func on NODE, which does not need